api/v1/db/json/notes-dao.js

```diff
@@ -214,5 +214,5 @@ const deleteNoteByID = noteID => new Promise((resolve, reject) => {
 });
 
 module.exports = {
-  getNotes, postNotes, patchNoteByID, deleteNoteByID,
+  getNotes, postNotes, getNoteByID, patchNoteByID, deleteNoteByID,
```

Comment:
I think this should use separate lines for each items.

Reply:
This should be fixed.